typescript-decorators/src/typescript-decorators.ts

```diff
@@ -1,19 +1,16 @@
+// Import reflect metadata for dependency injection mechanism
+import 'reflect-metadata'
```

Comment:
Is this really necessary? The migration guide does not import `reflect-metadata`:
https://github.com/L2jLiga/fastify-decorators/blob/next/lib/docs/Migration-to-v2.md

Reply:
`reflect-metadata` is necessary for DI mechanism. It provides ability to inject class annotated with `@Service` decorator via constructor params